impl/src/main/java/org/jboss/cdi/tck/tests/full/context/passivating/CityInterface.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.context.passivating;
+package org.jboss.cdi.tck.tests.full.context.passivating;

```

Comment:
If we remove `org.jboss.cdi.tck.tests.full.context.passivating.UnderwaterCity`, I believe we should remove `org.jboss.cdi.tck.tests.full.context.passivating.CityDecorator` and `org.jboss.cdi.tck.tests.full.context.passivating.CityInterface` as well.

Reply:
+1

Reply:
Removed.